CryptoTechProject/DeliveryMechanism.cs

```diff
@@ -22,11 +22,11 @@ public void Run(Action onStarted)
                 HttpListenerResponse response = context.Response;
 
                 //HardCodedWorkshopsGateway gateway = new HardCodedWorkshopsGateway();
-                AirtableGateway gateway = new AirtableGateway(System.Environment.GetEnvironmentVariable("AIRTABLE_URL"),
+               /*AirtableGateway gateway = new AirtableGateway(System.Environment.GetEnvironmentVariable("AIRTABLE_URL"),
                     System.Environment.GetEnvironmentVariable("AIRTABLE_API_KEY"),
-                    System.Environment.GetEnvironmentVariable("AIRTABLE_TABLE_ID"));
+                    System.Environment.GetEnvironmentVariable("AIRTABLE_TABLE_ID"));*/
                 
-                //AirtableGateway gateway = new AirtableGateway("https://api.airtable.com/", INSERT_API_KEY, INSERT_TABLE_ID);
+                AirtableGateway gateway = new AirtableGateway("https://api.airtable.com/", "keyZqU8PXE97s45Jw", "apppGaPfIFW8nmf4T");
```

Comment:
🚨is this your airtable api key?
You should take a look into using environment variables to hide these. Especially on a public repo!

Reply:
Yeah, I'm so sorry :< Keep on forgetting it cause was testing locally, it's rotated now